weave/tests/test_weave_client.py

```diff
@@ -753,8 +817,8 @@ def test_obj_dedupe(client):
     client._save_object({"a": 2}, "my-obj")
     res = client._objects()
     assert len(res) == 2
-    assert res[0].version_index == 0
-    assert res[1].version_index == 1
+    assert res[0].version_index in [0, 1]
```

Comment:
why is this needed? seems like the order is important?

Reply:
1. is it important? 
2. The sqlite primary key addition changed the ordering (i think)

we could add ordering to the query

@tssweeney

Reply:
might want to add the ordering to match CH